flows/engine/engine.go

```diff
@@ -95,20 +97,6 @@ func ResumeFlow(env flows.FlowEnvironment, run flows.FlowRun, event flows.Event)
 	return run.Session(), nil
 }
 
-// initializes our context based on our flow and current context
-func initTranslations(run flows.FlowRun) {
-	// set our language based on our contact if we have one
-	contact := run.Contact()
-	if contact != nil {
```

Comment:
I don't think there's anything in the engine that currently requires a contact, at least I've tried to make that not a requirement. Seems that might be a nice thing to maintain no?